subsys/spm/spm.c

```diff
@@ -567,3 +568,60 @@ void spm_config(void)
 	spm_config_sram();
 	spm_config_peripherals();
 }
+
+struct spm_uart_config {
+	const struct device *uart;
+	NRF_UARTE_Type *regs;
+};
+
+#ifdef CONFIG_UART_0_NRF_UARTE
```

Comment:
Shouldn't all of this be conditionally included only if you need to dump the non secure logs?